app/league/slack_messenger.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""Slack integration module."""
+import requests
+
+class SlackMessenger(object):
+
+    def __init__(self, app=None):
+        if app:
+            self.app = app
+            self.init_app(app)
+
+    def init_app(self, app):
+        self.url = app.config.get('SLACK_WEBHOOK')
+        self.channel = app.config.get('SLACK_CHANNEL')
+        self.payload = {'username': 'webhookbot',
+                        'icon_emoji': ':ghost:',
+                        'channel': self.channel}
+        app.extensions['messenger'] = self
+
+        def f(msg):
```

Comment:
Actually, since we don't want to fail completely if slack isn't configured, it'd be best to simply log a warning to `app.logger`